File: E-Estate/src/app/_model/company.ts
import { Estate } from "./estate";

export interface Company{
    id: number,
    companyName: string,
    address1: string,
    address2: string,
    address3:string,
    postcode:string,
    town:string,
    phone:string,
    email:string,
    fax:string,
    contactNo:string,
    managerName:string,
    isActive:boolean,
    estates: Estate[],
    townEstate:string,
    membershipType:string
}

File: E-EstateV2 API/E-EstateV2 API/Models/Company.cs
using System.ComponentModel.DataAnnotations;

namespace E_EstateV2_API.Models
{
    public class Company
    {
        [Key]
        public int Id { get; set; }
        public string companyName { get; set; }
        public string address1 { get; set; }
        public string address2 { get; set; }
        public string address3 { get; set; }
        public string postcode { get; set; }
        public string town { get; set; }
        public string phone { get; set; }
        public string email { get; set; }
        public string fax { get; set; }
        public string contactNo { get; set; }
        public string managerName { get; set; }
        public bool isActive { get; set; }

    }
}
